drf_api_logger/utils.py

```diff
@@ -76,3 +77,17 @@ def mask_sensitive_data(data, mask_api_parameters=False):
             data[key] = [mask_sensitive_data(item) for item in data[key]]
 
     return data
+
+def get_view_from_request(request):
+    try:
+        return f"{resolve(request.path_info)._func_path}"
```

Comment:
I think truncating this into 50 chars will be safer!

Reply:
Solved this bug, thanks for flagging this out. 

Do you have any other ideas? I would love to discuss and contribute.